examples/python-guide/advanced_example.py

```diff
@@ -50,7 +50,7 @@
 gbm = lgb.train(params,
                 lgb_train,
                 num_boost_round=10,
-                valid_sets=lgb_train,  # eval training data
+                valid_sets=lgb_eval,  # eval training data
```

Comment:
Thanks for the contribution! But this line is not a mistake. For the first training in this example, the goal is to print evaluation results on the training data to get an idea of how the fit is improving at each iteration.

The comment `# eval training data` was left here to try to say "yes, it's intentional that the training set is being used".

Do you think there is some rewording of that comment that would make it clearer?